src/model/rootelement.js

```diff
@@ -58,10 +58,11 @@ export default class RootElement extends Element {
 	 * @inheritDoc
 	 */
 	is( type, name ) {
+		const cutType = type.replace( 'model:', '' );
```

Comment:
Missing blank line after var def.

Reply:
Also, what's "cut type"?

Reply:
It's type without `module:` at the beginning to simplify logic in further lines.
Statement like:
```
( ( type == 'rootElement' || type == 'module:rootElement'  ) && name == this.name ) || super.is( type, name );
```
seems to be a little to complex in my opinion.